instagram/src/Components/PostContainer/Likes.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+class Likes extends React.Component{
+constructor(props){
+    super(props);
+    this.state= {
+        likes: props.likes,
+        likeToggle:false
```

Comment:
Nice use of a toggle flag so that a user can only always make one like at a time.